### ISSUE TITLE
Rerun tests using different MQ Size

### ISSUE THREAD

**Issue by araskachoi** [whiteblock/gossipsub-testing#15]
- edit the queue size of gossipsub
- the value is hardcoded at 32
- try with a queue size of 128, and with a queue size of 256

NOTE:
PL has also merged the pubsub tracer recently (https://github.com/libp2p/go-libp2p-pubsub/pull/227) and created a bunch of utilities for collecting, manipulating, and inspecting traces from a network (https://github.com/libp2p/go-libp2p-pubsub-tracer)

Should perform additional runs with/without tracing, and sending the dumps over so the libp2p team can inspect them.

**Comment by araskachoi** [comment]
@jasonatran the best way would be to fork [go-libp2p-pubsub](https://github.com/libp2p/go-libp2p-pubsub/blob/01b9825fbee1848751d90a8469e3f5f43bac8466/pubsub.go) to make the necessary changes and pull the code into our fork of adam's code in my [repo](https://github.com/araska-dev/go-libp2p-pubsub-benchmark-tools)

what do you think?

**Comment by jasonatran** [comment]
I think the tracer looks promising, so we can avoid any fork of `go-libp2p-pubsub` for now. We can revisit this after the first attempt to test using just the new tracer implementation. They already merged [go-libp2p-pubsub/pull/227](https://github.com/libp2p/go-libp2p-pubsub/pull/227) for us.

**Comment by araskachoi** [comment]
yes, but looks like that's going to require additional implementation into adam's code. for the sake of time, might be more efficient to run the tests with the changed message queue size first and then implement `tracer` later? or hold off on all tests until we get the `tracer`?

**Comment by jasonatran** [comment]
Oh yea, I do think we should fork `go-libp2p-pubsub-benchmark-tools`. We have a bunch of changes coming up for `tracer`, so I think it'll give us more agility if we fork off. I think a single run to verify that the queue size change fixes the dropped packet will be sufficient. My gut tells me it won't change the overall statistics of the gossip tests. If there is a change, we'll see it when we start testing with `tracer`.

**Comment by araskachoi** [comment]
will not use tracer for these tests and will move forward with using an increased messagequeuesize

https://github.com/araskachoi/go-libp2p-pubsub-benchmark-tools/blob/develop/pkg/host/host.go#L235
